frontend/src/utils/segment.ts

```diff
@@ -1,12 +1,12 @@
 export default class Segment {
-  authored: boolean;
+  knownAuthor?: string;
```

Comment:
```suggestion
  knownAuthor: string | null;
```
Since `knownAuthor` is always passed into the constructor but can be `null`, would it be better for `knownAuthor` to not be an optional attribute, but either `string` or `null`?

Reply:
Yes, I thought both of these were equivalent, thanks for pointing it out!